eZ/Bundle/EzPublishCoreBundle/ApiLoader/RepositoryFactory.php

```diff
@@ -98,6 +100,7 @@ public function buildRepository(PersistenceHandler $persistenceHandler, SearchHa
                     'policyMap' => $this->policyMap,
                 ),
                 'languages' => $this->configResolver->getParameter('languages'),
+                'content' => ['default_version_archive_limit' => $config['options']['default_version_archive_limit']],
```

Comment:
I think just `version_archive_limit` should be enough, otherwise anything one provides here will be the default :)